### PR TITLE
Scanpy scripts

### PR DESCRIPTION
**Pull request by nh3**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by pcm32** [comment]
Hi @pkrog! do you know who might be able to restart CI on this PR? Thanks!

**Comment by pkrog** [comment]
Hi @pcm32 , I think @bgruening just did it !

File: recipes/scanpy-scripts/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+package:
+  name: scanpy-scripts
+  version: {{ version }}
+
+source:
+  url: https://github.com/ebi-gene-expression-group/scanpy-scripts/archive/v{{ version }}.tar.gz
+  sha256: 4d9d3193857e5f801540236db547aa91db6c1db47c43943915f130c4fa8a31b2
+
+build:
+  number: 0
```
[the remark this answers is not in the document]
Comment (nh3) [reply recipes/scanpy-scripts/meta.yaml]: Yes, it's pure python, haven't tested it on 3.7 yet, but should be fine. I removed the noarch flag according to:

> Update 7 Feb 2018: Previously we had recommended that if the recipe is a pure Python package, it should be marked as a “noarch” package (details). However due to technical incompatibilies we can’t do this – so please DO NOT use "noarch" for now.

Is it obsolete now?

I'll have one more check to see if lower version of pandas and matplotlib works then it's good to merge.

[the remark this answers is not in the document]
Comment (bgruening) [reply recipes/scanpy-scripts/meta.yaml]: Yes sorry, this is outdated.

[the remark this answers is not in the document]
Comment (nh3) [reply recipes/scanpy-scripts/meta.yaml]: @bgruening py37 would only work with matplotlib>=3.0, where py36 work with matplotlib>=2.2. scanpy uses `skip: True # [not py36]`. It probably makes sense to be consistent with it, so I am not doing `noarch: generic` for now. Is it ok?

[the remark this answers is not in the document]
Comment (nh3) [reply recipes/scanpy-scripts/meta.yaml]: Strangely, `matplotlib>=2.2.0` brought back the "Failed to import any qt binding" error in mulled test, while it passed on my "local" mulled test. Bump the requirement back to `matplotlib>=3.0.0` and declare `noarch` this time.

**Comment by nh3** [comment]
Hi @bgruening, all tests have passed and I've replied to #1. Could you add me to the team so that I can merge this PR? Thank you!

**Pull request merged**